source/sass_api_reference/index.html.md.erb

```diff
@@ -0,0 +1,73 @@
+---
+title: Sass API Reference
+weight: 65
+---
+
+# Sass API Reference
```

Comment:
Since the markdown generates anchor links and classes I figured using markdown is the most maintainable way for the future, with the compromise of not being able to nest logic.